src/govuk/components/button/template.njk

```diff
@@ -37,7 +37,7 @@ treat it as an interactive element - without this it will be
 
 {#- Define common attributes we can use for both button and input types #}
 
-{%- set buttonAttributes %}{% if params.name %} name="{{ params.name }}"{% endif %}{% if params.disabled %} disabled="disabled" aria-disabled="true"{% endif %}{% if params.preventDoubleClick %} data-prevent-double-click="true"{% endif %}{% endset %}
+{%- set buttonAttributes %}{% if params.name %} name="{{ params.name }}"{% endif %}{% if params.disabled %} disabled aria-disabled="true"{% endif %}{% if params.preventDoubleClick %} data-prevent-double-click="true"{% endif %}{% endset %}
```

Comment:
potentially breaking change

button.disabled will still be true
but button.getAttribute('disabled') will change from "disabled" to and empty string

Reply:
Good catch Nick, thanks.

I'm going to split this change out into a follow up PR as it's the only change to the HTML output of one of the macros.

We're still discussing as a team how we want to treat the change, but at the very least it should go in the release notes, which will be easier once it's split out.

Reply:
Force-pushed to replace the changes in fcd04cd5a9bc2b8122cce5178d69ebf702104424 with 500a02fd2768270726c0658a1ac5608977445ade.